src/plugin/transform.ts

```diff
@@ -115,7 +115,7 @@ export function transform({
             containerClassNames: containerClasses,
             embedContentTransformFn: (raw) => {
                 const deprecatedHeadContent = concatStylesIncludeDirectives(
-                    `<base target="${baseTarget}">`,
+                    `<base target="${baseTarget}"><meta http-equiv="Content-Security-Policy" content="script-src 'none'">`,
                     styles,
```

Comment:
maybe
meta `prop` with default `<meta http-equiv="Content-Security-Policy" content="script-src 'none'">`

Reply:
No, because we set the entire head content with the current setting, and individual settings have been deprecated. If you add a separate property to add some value to the head, it will make the development of the plugin properties inconsistent.